torchtune/models/gemma/_model_builders.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import List, Optional
+from functools import partial
+
+from torch import nn
+
+from torchtune.models.gemma._component_builders import gemma, lora_gemma
+
+from torchtune.modules import Tokenizer, TransformerDecoder
+from torchtune.modules.peft import LORA_ATTN_MODULES
+
+"""
+Model builders build specific instantiations using component builders. For example
+the ``gemma_2b`` model builder uses the ``gemma`` component builder.
+"""
+
+
+def gemma_2b() -> TransformerDecoder:
+    """
+    Builder for creating a Gemma 2B model initialized w/ the default 2b parameter values
+
+    Returns:
+        TransformerDecoder: Instantiation of Gemma 2B model
+    """
+    return gemma(
+        vocab_size=256_000,
```

Comment:
Still shocked by this vocab size - so large!

Reply:
Yeah, :joy:

Reply:
Does this mean the embedding(/output projection since they're tied) constitutes a full 25% of their params?!

Reply:
I calculate it with `count_trainable_parameters `, the params of embed_tokens is 21%.

```
def count_trainable_parameters(model):
    return sum(p.numel() for p in model.parameters() if p.requires_grad)
```